recipe/meta.yaml

```diff
@@ -23,8 +23,13 @@ build:
     - mscolab = mslib.mscolab.mscolab:main
 
 requirements:
+  build:
+    - python
+    - pip 
+    - setuptools ==65.5.1
   host:
     - python
+    - setuptools ==65.5.1
```

Comment:
Repodata of `conda-build` was patched to constrain `setuptools` to `<66` recently ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/387 ). So would be curious to know why this was needed? Is there more context somewhere?

Reply:
I iterated this, see here, this installs  66.1.1-pyhd8ed1ab_0

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=649832&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412&l=849

im my case refactoring our version number will solve it at all, this was proven by @zklaus